### PR TITLE
Homepage design follow ups

### PR DESCRIPTION
**Pull request by muhajirdev**

### CODE REVIEW

**Comment by mmahalwy** [comment]
#### 📝 Changed routes:


| [**/**](https://quran-com-8u7bjdmd5-qurancom.vercel.app/) |
|:-:|
|<a href="https://quran-com-8u7bjdmd5-qurancom.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/sRUXxiE_gcioev1PWC7h5HnVKQvBFcwbHgxJ150YdMFeBdnxvPQrzy9faDHHxDoIvMtSWytF72EieNZKRYZt0A.png">(view full size)</a> |







Commit d3aac235c75604bbe47dd193ce5128f8b9fffd41 (https://quran-com-8u7bjdmd5-qurancom.vercel.app).

File: src/components/HomePage/HomePageHero.module.scss [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+}
+
+.backgroundImage {
+  position: absolute;
+  z-index: -1;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-image: url(../../../public/images/background.jpg);
```
Comment (tifa2UP) [inline src/components/HomePage/HomePageHero.module.scss:15]: prefer absolute path. /public/images

Comment (tifa2UP) [reply src/components/HomePage/HomePageHero.module.scss:15]: also, the url should be in quotes? or is this some webpack magic

File: src/components/HomePage/HomePageHero.module.scss [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+  background-size: cover;
+  @include theme.dark {
```
Comment (tifa2UP) [inline src/components/HomePage/HomePageHero.module.scss:17]: Does handle OS dark theme?

**Review by tifa2UP — APPROVED** [review]

**Pull request merged**